pages.de/common/git-send-email.md

```diff
@@ -0,0 +1,24 @@
+# git send-email
+
+> Sende Patches als Email. Möglich für einzelene Commits, Ordner oder .patch Datein.
```

Comment:
Seems to not be an equal translation of the English page ( the second sentence).
English version:
`Send a collection of patches as emails. Patches can be specified as files, directions, or a revision list.`

Reply:
```suggestion
> Sende eine Menge von Patches als E-Mail. Patches können als Dateien, Ordner oder Liste von Revisionen spezifiziert werden.
```

This translation is closer